private String showFailures() throws Exception {
    if (theInstance.deleteFailures.size() == 1) {
        return "[Delete Failed: " + theInstance.deleteFailures.get(0) + "]";
    } else {
        return "[Delete Failures: " + theInstance.deleteFailures.size() + "]";
    }
}